wordpress-via-iac/completed/data.tf

```diff
@@ -0,0 +1,43 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+data "aws_ami" "al2" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-kernel-5.10-hvm-2.0.20230320.0-x86_64-gp2"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["137112412989"] # Amazon
+}
+
+
+# data "aws_secretsmanager_secret" "wordpress_rds_admin" {
+#   name = aws_secretsmanager_secret.wordpress_rds_admin.name
+#   depends_on = [
+#     aws_secretsmanager_secret.wordpress_rds_admin
+#   ]
+# }
+
+# data "aws_secretsmanager_secret_version" "wordpress_rds_admin" {
+#   secret_id = data.aws_secretsmanager_secret.wordpress_rds_admin.id
+# }
+
+
+# data "aws_secretsmanager_secret" "wordpress_rds_password" {
+#   name = aws_secretsmanager_secret.wordpress_rds_password.name
+#   depends_on = [
+#     aws_secretsmanager_secret.wordpress_rds_password
+#   ]
+# }
+
+# data "aws_secretsmanager_secret_version" "wordpress_rds_password" {
+#   secret_id = data.aws_secretsmanager_secret.wordpress_rds_password.id
+# }
```

Comment:
Why's this commented out?  If it's 'to use later' then place it somewhere in the guide, to be moved into the file, rather than _uncomment this line_.

Reply:
Not needed at all - removed